jobs/openshift/cluster/integreatly/test/Jenkinsfile

```diff
@@ -0,0 +1,109 @@
+  testOptions.adminUsername = 'integreatly'
+  testOptions.adminPassword = 'Password1'
+  testOptions.awsRegion = 'eu-west-2'
+  testOptions.awsAccountName = 'fheng.AWS'
+  testOptions.clusterName = params.installerGitBranch
+  testOptions.domainName = 'skunkhenry.com'
+  testOptions.gitUrl = params.installerGitUrl
+  testOptions.gitBranch = params.installerGitBranch
+  testOptions.openshiftMasterUrl = "https://${params.installerGitBranch}.${testOptions.domainName}"
+  verifyParameters(testOptions)
+}
+
+node('cirhos_rhel7') {
+  cleanWs()
+  stage('Create Cluster') {
+    println "[INFO] Creating ${testOptions.clusterName} cluster"
+    def jobName = 'openshift-cluster-create'
+    def instanceType = 'm4.xlarge'
+    def instanceGroupSize = '3'
+    def jobParams = [
+      [$class: 'StringParameterValue', name: 'clusterName', value: testOptions.clusterName],
+      [$class: 'StringParameterValue', name: 'awsRegion', value: testOptions.awsRegion],
+      [$class: 'StringParameterValue', name: 'awsAccountName', value: testOptions.awsAccountName],
+      [$class: 'StringParameterValue', name: 'master_instance_type', value: instanceType],
+      [$class: 'StringParameterValue', name: 'compute_instance_type', value: instanceType],
+      [$class: 'StringParameterValue', name: 'infra_instance_type', value: instanceType],
+      [$class: 'StringParameterValue', name: 'master_group_size', value: instanceGroupSize],
+      [$class: 'StringParameterValue', name: 'compute_group_size', value: instanceGroupSize],
+      [$class: 'StringParameterValue', name: 'infra_group_size', value: instanceGroupSize],
+      [$class: 'BooleanParameterValue', name: 'dryRun', value: params.dryRun]
+    ]
+    buildJob(jobName, jobParams)
+  }
+
+  stage('Install Integreatly') {
+    println "[INFO] Installing Integreatly in ${testOptions.clusterName} cluster"
+    def jobName = 'openshift-cluster-integreatly-install'
+    def userCount = '50'
+    def selfSignedCerts = false
+    def jobParams = [
+      [$class: 'StringParameterValue', name: 'clusterName', value: testOptions.clusterName],
+      [$class: 'StringParameterValue', name: 'openshiftMasterUrl', value: testOptions.openshiftMasterUrl],
+      [$class: 'StringParameterValue', name: 'clusterAdminUsername', value: testOptions.adminUsername],
+      [$class: 'StringParameterValue', name: 'clusterAdminPassword', value: testOptions.adminPassword],
+      [$class: 'StringParameterValue', name: 'installerGitUrl', value: testOptions.gitUrl],
+      [$class: 'StringParameterValue', name: 'installerGitBranch', value: testOptions.gitBranch],
+      [$class: 'StringParameterValue', name: 'userCount', value: userCount],
+      [$class: 'BooleanParameterValue', name: 'selfSignedCerts', value: selfSignedCerts],
+      [$class: 'BooleanParameterValue', name: 'dryRun', value: params.dryRun]
+    ]
+    buildJob(jobName, jobParams)
+  }
+
+  stage('Test Integreatly') {
+    println "[INFO] Testing Integreatly"
```

Comment:
Jira on the backlog to cover implementing this stage https://issues.jboss.org/browse/INTLY-1700. Will probably need sync up with @tremes et al on what integration testing is in place already.

Reply:
The problem with testing is that we heavily rely on admin permissions. I've heard about some dedicated-admin account, but it still doesn't sound much promising.

Reply:
@tremes OK thanks. We will need to run integration testing since it's one of the requirements. If you have any comments/suggestions/thoughts about any of that, please add it into that ticket.

Reply:
I was obviously confused here. I thought we were talking about the OSD testing. I think we should be OK with the clusters provisioned by ansible tower tooling since there's admin account available.